### DISCUSSION TITLE
VAC worries

### DISCUSSION THREAD

**Discussion by Shadowysn** [Joshua-Ashton/VPhysics-Jolt#158] [General]
Paranoid person here.
This seems to replace **vphysics.dll** and I'm worried if there's the possibility it can trigger VAC.

**Comment by misyltoad** [comment]
Then play in -insecure or single player.

**Comment by Shadowysn** [comment]
Thanks for informing.
Though if that's the case then I'd suspect a few uninformed and unexperienced people may risk it unknowingly without such a disclaimer. 😓

**Reply by misyltoad** [reply]
VAC only bans cheaters

**Reply by Shadowysn** [reply]
> VAC only bans cheaters

I suppose, but doesn't the client's vphysics get used for client-side props on servers? What's to stop VAC from detecting the altered dll in that case?

**Reply by Ristovski** [reply]
From my understanding (and what I could gather online), VAC in Garrys Mod seems to be disabled. So as long as you use it in gmod, you **should** be fine.

**Comment by SudoStdUser** [comment]
not join in servers with on VAC
